action/action.go

```diff
@@ -9,6 +9,9 @@ const (
 	// addAction defines the action for creating a resource.
 	addAction = "add"
 
+	// cancelAction defines the action for changing ownership of a resource.
```

Comment:
I think this comment needs to be updated:

```suggestion
	// cancelAction defines the action for canceling a resource.
```

Reply:
good catch. corrected.